frontend/src/User/Shoutouts/ShoutoutForm/ShoutoutForm.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useContext, useState } from 'react';
+import { Form, TextArea, Segment, Label, Button } from 'semantic-ui-react';
+import { UserContext } from '../../../UserProvider/UserProvider';
+import CustomSearch from '../../../Common/Search/Search';
+import Emitters from '../../../EventEmitter/constant-emitters';
+import { Member, MembersAPI } from '../../../API/MembersAPI';
+
+type Shoutout = {
+  shoutoutGiverEmail: string;
+  shoutoutRecipientEmail: string;
+  message: string;
+};
+
+const ShoutoutForm: React.FC = () => {
+  const userEmail = useContext(UserContext).user?.email;
+  const [members, setMembers] = useState<IdolMember[] | undefined>(undefined);
+  const [recipient, setRecipient] = useState<IdolMember | undefined>(undefined);
+  const [message, setMessage] = useState('');
+
+  MembersAPI.getAllMembers().then((mems) => {
+    setMembers(mems);
+  });
+
+  const sendShoutout = () => {
+    if (!recipient) {
+      Emitters.generalError.emit({
+        headerMsg: 'No Member Selected',
+        contentMsg: 'Please select a member!'
+      });
+    } else if (recipient && userEmail && message !== '') {
+      const shoutout: Shoutout = {
+        shoutoutGiverEmail: userEmail,
+        shoutoutRecipientEmail: recipient.email,
+        message
+      };
+      console.log(shoutout);
+      alert('Shoutout successfully submitted!');
```

Comment:
You can probably suppress this warning, or just disable all linter warnings for alert.